3Dmol/glmodel.js

```diff
@@ -1445,15 +1445,22 @@ $3Dmol.GLModel = (function() {
          */
         this.getCrystData = function() {
             if (modelData.cryst) {
+                // add the matrix if it is missing
+                if (!modelData.cryst.matrix) {
+                    const cryst = modelData.cryst;
+                    modelData.cryst.matrix = $3Dmol.conversionMatrix3(
```

Comment:
Should I rename `conversionMatrix3` to something like `createCrystMatrix` or similar?

Reply:
Nah, that's an even narrower name for a somewhat general function.